aspnetcore/fundamentals/servers/kestrel.md

```diff
@@ -253,6 +289,49 @@ You can configure the rates per request in middleware:
 
 ::: moniker-end
 
+::: moniker range=">= aspnetcore-2.2"
+
+**MaxStreamsPerConnection**
+
+`Http2.MaxStreamsPerConnection` limits the number of concurrent request streams per HTTP/2 connection. Excess streams are refused.
+
+```csharp
+.ConfigureKestrel((context, options) =>
```

Comment:
Indicate where this code belongs.

Reply:
I have the `ConfigureKestrel` call shown directly under the `Program.Main` (near where the topic calls out `UseKestrel` with options, too) much earlier in the topic (up around Lines 114-125). The next 11 appearances of `ConfigureKestrel` follow this format, too, to minimize the size of the examples.

I originally noted more work for this in the 2.2 master tracking issue: #7180. I've just opened a proper issue to update the sample at #8571.